book/src/command_line.md

```diff
@@ -12,4 +12,15 @@ $ flutter_rust_bridge_codegen --help
 ```shell
 $ dart run flutter_rust_bridge_serve --help
 {{#include help.serve.txt}}
-```
\ No newline at end of file
+```
+
+## Configuration files
+
+You can also run `flutter_rust_bridge_codegen` with no arguments, provided one of these files exist (in order of priority):
+
+- `.frb.yml`
+- `.frb.yaml`
+- `.frb.json`
```

Comment:
uber-nit: shall we use `flutter_rust_bridge` instead of `frb`, because when a reader reads "frb" in a repo, he may only google "frb" or "frb github" and never finds flutter_rust_bridge